From linux-kernel-owner+greg=40kroah.com-S1161073AbWAGXo5@vger.kernel.org Sat Jan  7 15:48:06 2006
Message-ID: <43C0524F.1030602@trash.net>
Date: Sun, 08 Jan 2006 00:44:15 +0100
From: Patrick McHardy <kaber@trash.net>
To: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
Subject: W1: Remove incorrect MODULE_ALIAS

The w1 netlink socket is created by a hardware specific driver calling
w1_add_master_device, so there is no point in including a module alias
for netlink autoloading in the core.

Signed-off-by: Patrick McHardy <kaber@trash.net>
Acked-by: Evgeniy Polyakov <johnpol@2ka.mipt.ru>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/w1/w1_int.c |    2 --
 1 file changed, 2 deletions(-)

--- gregkh-2.6.orig/drivers/w1/w1_int.c
+++ gregkh-2.6/drivers/w1/w1_int.c
@@ -208,5 +208,3 @@ void w1_remove_master_device(struct w1_b
 
 EXPORT_SYMBOL(w1_add_master_device);
 EXPORT_SYMBOL(w1_remove_master_device);
-
-MODULE_ALIAS_NET_PF_PROTO(PF_NETLINK, NETLINK_W1);
